src/docs/development/ios-project-migration.md

```diff
@@ -10,7 +10,15 @@ description: How to migrate existing Flutter iOS projects to Xcode 11.4.
 
 To develop Flutter apps for iOS, you need a Mac with Xcode installed.
 Xcode 11.4 changed the way frameworks are linked and embedded, and
-you may [experience issues switching between iOS devices and simulators][].
+you may [see the following errors when switching between iOS devices and simulators:][]
```

Comment:
doesn't this text have to match the link at the bottom (line 50)?

Reply:
I think @christopherfujino is correct. Otherwise, looks good to me! Thanks for the speedy response to my lack of search-fu!

Reply:
sigh. yes. thanks for noticing.

Reply:
(the sigh comment was directed at @christopherfujino for noticing the link issue, not at @kf6gpe 😃 )

Reply:
@jmagman, another trick you might like better. You can do this:

```
you may [see the following errors when switching between iOS devices and simulators:][errors]

[errors]: URL
```

Putting a short-cut in the second pair of otherwise empty brackets, then put the shortcut in the list of URL references.

Reply:
ooo nice, done.